lib/frameworks/custom_jasmine.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
This file should be in the spec folder. I'd put it at `spec/custom/framework.js`